lib/blocktree/blocktree.go

```diff
@@ -16,11 +17,14 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promauto"
 )
 
-var leavesGauge = promauto.NewGauge(prometheus.GaugeOpts{
-	Namespace: "gossamer_block",
-	Name:      "leaves_total",
-	Help:      "total number of blocktree leaves",
-})
+var (
+	leavesGauge = promauto.NewGauge(prometheus.GaugeOpts{
+		Namespace: "gossamer_block",
+		Name:      "leaves_total",
+		Help:      "total number of blocktree leaves",
+	})
+	errorAncestorOutOfBoundsCheck = errors.New("out of bounds ancestor check")
```

Comment:
supernit not sure why but convention seems to be `errXXX` not `errorXXX`... but really I feel like this convention I got used to for years is retarded... any thoughts anyone? Should we prefix our errors with.... `error` instead of `err`/`Err`? :thinking: Not a big deal either of course

Reply:
I just kept with error